casper-event-standard/Cargo.toml

```diff
@@ -12,15 +12,15 @@ license.workspace = true
 doctest = false
 
 [dependencies]
-casper-types = "3.0.0"
-casper-event-standard-macro = { version = "0.4.0", path = "../casper-event-standard-macro" }
+casper-types = { git = "https://github.com/casper-network/casper-node.git", default-features = false }
```

Comment:
We should not publish this to crates.io. This will cause problems with `casper-types` versions miss-match all over the places. My plan is to release next version of CES after Casper 2.0 is released and we can point at stable `casper-types`.

If you'd like to have this code in this repo, I can make a new branch for it: `casper-2.0` that will point at git. In Odra we use `feat-2.0` branch.

Reply:
Indeed I need this create to dev ceps please. Can we please have a branch witth the correct crates and updates for current dev of Casper node. I would have removed the git on release when core is ready and has released new crates to release this crate.

Yes please I would like this code in the repo, may you want to do a specific branch is also fine, be aware those feat-2.0 have  been merged on core and not used anymore. There is no real need for this feat-2.0 branches now but if you want then I'll rebase. Can you then please branch the develop branch then so that I rebase this PR to it. Thanks.